app/src/main/kotlin/com/simplemobiletools/draw/pro/activities/MainActivity.kt

```diff
@@ -335,6 +345,16 @@ class MainActivity : SimpleActivity(), CanvasListener {
         }
     }
 
+    private fun eyeDropperClicked() {
+        isEyeDropperOn = !isEyeDropperOn
+        if (isEyeDropperOn) {
+            eyeDropper.start()
+        } else {
+            eyeDropper.stop()
+        }
+        eye_dropper.setImageDrawable(ContextCompat.getDrawable(this, if (isEyeDropperOn) R.drawable.ic_colorise_off_vector else R.drawable.ic_colorize_vector))
```

Comment:
it is "colorize", not "colorise". Also move the if/else into some variable, this like has way too many things going on